src/main/resources/templates/db_scripts/uuidFunctions.sql

```diff
@@ -3,3 +3,9 @@ BEGIN
 <#include "uuid.ftl">
 EXCEPTION WHEN OTHERS THEN NULL;
 END $uuidftl$;
+
+DO $$ BEGIN DROP FUNCTION count_estimate_smart; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN DROP FUNCTION count_estimate_smart_depricated; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN DROP FUNCTION set_id_injson_actual_opening_hours; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN DROP FUNCTION set_id_injson_exceptional_hours; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN DROP FUNCTION set_id_injson_regular_hours; EXCEPTION WHEN OTHERS THEN END; $$;
```

Comment:
When testing the upgrade as explained in
https://issues.folio.org/browse/MODCAL-68?focusedCommentId=74254&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-74254
this doesn't work.
See #117 how to fix this trigger issue.
These functions are not related to uuid.ftl.
Can you move all *.sql scripts that do the upgrade from RMB 24 to RMB 29 into one file? See #117 for an example.

Reply:
@julianladisch  Thanks!